src/core/EdFi.DataManagementService.Core/EdFi.DataManagementService.Core.csproj

```diff
@@ -31,6 +32,12 @@
     <PackageReference Include="JsonPath.Net" Version="1.0.0" />
   </ItemGroup>
 
+  <ItemGroup>
+    <InternalsVisibleTo Include="EdFi.DataManagementService.Core.Tests.Unit" />
```

Comment:
With the move to `internal` to hide from frontends/backends, need to allow the unit test project to see them.